### PR TITLE
Added CI files

### PR DESCRIPTION
**Pull request by Flast**

### CODE REVIEW

**Comment by djowel** [comment]
wonderful!

**Comment by Flast** [comment]
Note that failure of bug4853 caused by boostorg/range#54

**Comment by Flast** [comment]
Hmm... Travis exceeds its running time limit, so we need breaking a job...

**Comment by Flast** [comment]
I reorganized (and temporary disabled some) lambda_tests, so I hope that CIs are passed.

File: test/scope/lambda_tests10.cpp [hunk 1]
```diff
@@ -15,6 +18,9 @@ struct zzz {};
 
 int main()
 {
+#if defined(RUNNING_ON_APPVEYOR) && BOOST_WORKAROUND(BOOST_MSVC, == 1700)
```
Comment (Flast) [inline test/scope/lambda_tests10.cpp]: Oops, it should be 1800 (1700 is msvc-11).

**Comment by Flast** [comment]
Yeah! Now all CIs are passed.

**Comment by djowel** [comment]
Wonderful!

**Pull request merged**